dot/services_integration_test.go

```diff
@@ -354,10 +364,30 @@ func TestNewWebSocketServer(t *testing.T) {
 }
 
 func Test_createPprofService(t *testing.T) {
-	t.Parallel()
-	settings := pprof.Settings{}
-	service := createPprofService(settings)
-	require.NotNil(t, service)
+	type args struct {
+		settings pprof.Settings
+	}
+	tests := []struct {
+		name string
+		args args
+		want *pprof.Service
```

Comment:
Change this to be i.e. `notNil bool` since you can't really assert the entire struct due to private fields.

Reply:
Updated.